tests/pull.bats

```diff
@@ -266,3 +266,46 @@ load helpers
   iid=$output
   run_buildah rmi ${iid}
 }
+
+@test "pull-policy" {
+  mkdir ${TESTDIR}/buildahtest
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy bogus alpine
+
+  #  If image does not exist the never will fail
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  run_buildah 125 inspect alpine
```

Comment:
I'll speak for @edsantiago, he'll probably want the error verbiage to be checked too.

Reply:
Yes please! Thank you @TomSweeneyRedHat, I haven't been monitoring this one.

Reply:
Added output tests.